vscode-extension/package.json

```diff
@@ -150,7 +153,31 @@
           }
         ]
       }
-    ]
+    ],
+    "menus": {
+      "editor/title": [
+        {
+          "command": "vscode-aiconfig.share",
+          "when": "resourcePath =~ /.*(\\.aiconfig\\.json|\\.aiconfig\\.yaml|\\.aiconfig\\.yml|\\.aiconfig)$/",
```

Comment:
Will the double backslashes work on both windows and mac?

Reply:
That's just an escape for the regex string, it shouldn't be specific to the OS, just the regex parser.

Reply:
We can validate this on windows